lib/dialects/postgres/query-generator.js

```diff
@@ -746,6 +746,9 @@ const QueryGenerator = {
 
   quoteIdentifier(identifier, force) {
     if (identifier === '*') return identifier;
+    if (identifier.indexOf('.') !== -1){
```

Comment:
To only check this if needed I'd move the condition into the already existing if. 
As in:
```js
if (!force && this.options && this.options.quoteIdentifiers === false && identifier.indexOf('.') === -1) { // default is `true`
```